serverless-logic/serverless-logic-getting-started/serverless-logic-creating-managing-workflows.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="serverless-logic-creating-managing-workflows"]
+= Creating and managing workflows
+:context: serverless-logic-creating-managing-workflows
+include::_attributes/common-attributes.adoc[]
```

Comment:
I am not sure if it matters (i.e, idk if it will break anything), but per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata) these two lines should be flipped. Same applies to other assemblies here.
```suggestion
include::_attributes/common-attributes.adoc[]
:context: serverless-logic-creating-managing-workflows
```